controller/space_iterations.go

```diff
@@ -115,8 +115,8 @@ func (c *SpaceIterationsController) Create(ctx *app.CreateSpaceIterationsContext
 // List runs the list action.
 func (c *SpaceIterationsController) List(ctx *app.ListSpaceIterationsContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
-		_, err := appl.Spaces().Load(ctx, ctx.SpaceID)
-		if err != nil {
+		exists, err := appl.Spaces().Exists(ctx, ctx.SpaceID.String())
+		if !exists || err != nil {
```

Comment:
I wanted to discuss about these things https://github.com/almighty/almighty-core/pull/1415#discussion_r122726636